modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -369,6 +382,19 @@ public function get_close_gutenberg_url() {
 		return $this->get_calypso_url();
 	}
 
+	/**
+	 * Returns the URL for switching the user's editor to the Calypso (WordPress.com Classic) editor.
+	 *
+	 * @return string
+	 */
+	public function get_switch_to_classic_editor_url() {
+		return add_query_arg(
+			'set-editor',
+			'classic',
+			$this->is_calypsoify_enabled ? $this->get_calypso_url( get_the_ID() ) : false
```

Comment:
I don't really understand why we are checking if Calypsoify is enabled here. The classic editor is available to everyone, not just Calypsoify users. Am I missing something here?

Reply:
You're right, this URL will only ever be seen in the case of `wpcomGutenberg.switchToClassic.isVisible` being `true`, which means we're in an iframed editor. It's more for the sake of keeping the code inline with dotcom's code, which we're going to have to reconcile at some point in the future.

Reply:
Note that in WP.com we show the button in both editors (iframed and WP Admin editor). I think we should do the same in JP/Atomic (specially for AT users), probably as part of https://github.com/Automattic/wp-calypso/issues/32047